File: examples-sdk/kotlin/001b-vilapp-ai-gw-benchmark/main.kt
// 001b-vilapp-ai-gw-benchmark — Kotlin SDK equivalent
// Compile: vil compile --from kotlin --input 001b-vilapp-ai-gw-benchmark/main.kt --release

fun main() {
    val server = VilServer("ai-gw-bench", 3081)
    val gw = ServiceProcess("gw")
    server.service(gw)
    server.compile()
}

File: examples-sdk/csharp/001b-vilapp-ai-gw-benchmark/Main.cs
// 001b-vilapp-ai-gw-benchmark — C# SDK equivalent
// Compile: vil compile --from csharp --input 001b-vilapp-ai-gw-benchmark/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("ai-gw-bench", 3081);
var gw = new ServiceProcess("gw");
server.Service(gw);
server.Compile();
